src/form/utils.py

```diff
@@ -54,7 +54,11 @@ def __is_int(value):
     return any(isinstance(value, i) for i in (int, np.int8, np.int16, np.int32, np.int64))
 
 def __is_float(value):
-    return any(isinstance(value, i) for i in (float, np.float16, np.float32, np.float64, np.float128))
+    try:
```

Comment:
Instead I would suggest to use a more deterministic approach. For example:

At the top of the file, add something like

```
SUPPORTED_FLOAT_TYPES = [float, np.float16, np.float32, np.float64]
if hasattr(np, "float128"):
    SUPPORTED_FLOAT_TYPES.append(np.float128)
```

then use `return any(isinstance(value, i) for i in SUPPORTED_FLOAT_TYPES)`